src/components/views/settings/tabs/user/LabsUserSettingsTab.tsx

```diff
@@ -180,7 +180,10 @@ export default class LabsUserSettingsTab extends React.Component<{}, IState> {
                     }
                 </div>
                 { betaSection }
-                { labsSections }
+                { SdkConfig.get("show_labs_settings") && <>
```

Comment:
we already gate this on line 95 - we should reuse that check instead of splitbraining the logic a bit.